src/System.Private.Xml/src/System/Xml/Core/RawTextWriter.ttinclude

```diff
@@ -0,0 +1,44 @@
+﻿<#@ template debug="false" hostspecific="false" language="C#" #>
+<#
```

Comment:
I think extra enter might be coming from here. Try this out `#><#` insted of `#>\n<#`